### PR TITLE
fix: button and accordion controls not inheriting font-family styles and update fast packages

### PR DESCRIPTION
**Pull request by SethDonohue**
#### Pull request checklist

- [x] Include a change request file using `$ yarn change`

#### Description of changes

- Fixes a bug where the `button` element on the shadow DOM for the accordion-item and button components does not inherit the font-family as it should.
- Also updates fast dependencies

### CODE REVIEW

File: packages/web-components/package.json [hunk 1]
```diff
@@ -94,8 +94,8 @@
   "dependencies": {
     "@microsoft/fast-colors": "^5.1.0",
     "@microsoft/fast-components-styles-msft": "^4.29.0",
-    "@microsoft/fast-element": "^0.16.0",
-    "@microsoft/fast-foundation": "^1.7.0",
+    "@microsoft/fast-element": "^0.17.0",
+    "@microsoft/fast-foundation": "^1.7.1",
```
Comment (chrisdholt) [inline packages/web-components/package.json:98]: Did you run yarn after this update? I'd expect to see an update to the yarn lockfile here.

File: packages/web-components/package.json [hunk 2]
```diff
@@ -94,8 +94,8 @@
   "dependencies": {
     "@microsoft/fast-colors": "^5.1.0",
     "@microsoft/fast-components-styles-msft": "^4.29.0",
-    "@microsoft/fast-element": "^0.16.0",
-    "@microsoft/fast-foundation": "^1.7.0",
```
Comment (chrisdholt) [inline packages/web-components/package.json:98]: I think this *could* stay at 1.7.x - this will take 1.7.1 as acceptable.

Comment (SethDonohue) [reply packages/web-components/package.json:98]: I was curious about that, I'll switch it back to ^1.17.0

File: packages/web-components/package.json (hunk 1 above)
Comment (SethDonohue) [reply packages/web-components/package.json:98]: I did, but looks like I forgot to commit the lockfile!

**Comment by chrisdholt** [comment]
Thanks @SethDonohue!

**Pull request merged**